cassdegrees/cassdegrees/settings.py

```diff
@@ -102,12 +107,13 @@
     },
     {
         'NAME': 'django.contrib.auth.password_validation.MinimumLengthValidator',
+        'OPTIONS': {'min_length': 10}
     },
     {
         'NAME': 'django.contrib.auth.password_validation.CommonPasswordValidator',
```

Comment:
Are these other validators needed?

Reply:
Technically no, but some of them ensure the user doesn't enter in a password that has their username in it/other things like that. It's basically to make an attempt at following the privacy standards found here: https://policies.anu.edu.au/ppl/document/ANUP_013008

Specifically, it mentions the following:
- the account name in any form (as-is, reversed, capitalised, doubled, etc.);
- the user’s first or last name in any form;
- simple patterns of letters on keyboards; or
- any well-known or publicly posted identifiable information.

Technically all of these aren't checked for, but I think this is partially a protocol for users to follow so it might not be a huge issue. Having said that, I'll try and remove some of the ones that aren't strictly necessary

Reply:
Does this also apply to authentication then? If a weak password is setup for a user for whatever reason (i.e by manual database entry), this will mean they would be locked out, no?

I would understand this for the creation of passwords, however.

Reply:
This is only for password creation. Insecure password still work normally